Index.html

```diff
@@ -13,206 +12,222 @@
 <body>
```

Comment:
As highlighted by the previous reviewer, I suggest that you  fix your project to fit the entire width of the screen. This is your project currently
<img width="1650" alt="Screenshot 2022-11-04 at 08 33 29" src="https://user-images.githubusercontent.com/48546992/199898107-aeefa6f9-0ab4-4d57-ae24-caea92db5813.png">
<img width="1649" alt="Screenshot 2022-11-04 at 08 33 37" src="https://user-images.githubusercontent.com/48546992/199898121-f5d9ea54-0715-4a07-b754-2a56455c3f87.png">
I suggest you add add `margin:0` property to your body to override system default.
<img width="1644" alt="Screenshot 2022-11-04 at 08 26 32" src="https://user-images.githubusercontent.com/48546992/199898628-e74c5d16-7b50-4486-9a96-19af641df446.png">
As you can see in the above screenshot, the system has its own default which you will need to override by adding the properties to body